ironmon_tracker/data/RandomizerLog.lua

```diff
@@ -2,11 +2,12 @@
 RandomizerLog = {}
 
 RandomizerLog.Patterns = {
-	RandomizerVersion = "Randomizer Version:%s*([%d%.]+)%s*$", -- Note: log file line 1 does NOT start with "Rando..."
+	-- Acount for possible -dev or -beta in version string
+	RandomizerVersion = "Randomizer Version:%s*([%d%.]+).*$", -- Note: log file line 1 does NOT start with "Rando..."
 	RandomizerSeed = "^Random Seed:%s*(%d+)%s*$",
 	RandomizerSettings = "^Settings String:%s*(.+)%s*$",
 	RandomizerGame = "^Randomization of%s*(.+)%s+completed",
-	PokemonName = "([%u%d%.]* ?[%u%d%.'%-♀♂%?]+).-",
+	PokemonName = "(%u%a*%s?[%a%d%.’'%-♀♂ ]*%s?[%a%d%.’'%-♀♂ ]*)",
```

Comment:
The ironmon randomizer does not have fully capitalised pokemon names in the log file. Comparison between the regular and ironmon randomizer: 
![image](https://user-images.githubusercontent.com/19582161/221446633-398a2487-bb79-4f14-a21c-18e52257f2d4.png)
![image](https://user-images.githubusercontent.com/19582161/221446640-3b7cec43-7e75-4155-8c7b-0e7571490937.png)

Additionally the pattern was changed to be more generic, matching an uppercase letter, any number of following letters, an optional space, then 1-2 instances of the character set that can be found in pokemon names. Looking at this now I can trim it down a bit so I will make another commit to do so.

Reply:
Okay I feel like a real goober but its not specific to the ironmon randomizer, I just tend to tick "lower case pokemon names". Which would break the parsing

Reply:
A note on this pattern: I expect to be reworking the parser to accommodate different Randomizer settings and custom hacks to Pokémon names, moves, etc. You can read more about this in #357 

Until then, I'd like to hold off on altering this pattern. This pattern is very important and needs to be tested thoroughly. There are weird circumstances where sometimes the names that you see in your log aren't the same as the ones seen in other logs for other people. For example, sometimes `Nidoran ♂` appears as `Nidoran ?` which is why there is a whole separate set of functions to deal with parsing this information properly. I see that your updated pattern lacks the `%?` and may not properly capture these cases.

I'm willing to make changes here, but only if they have been thoroughly tested and only if you can explain in detail the need for each one, and what they accomplish. With that said, don't feel the need to do this as I expect I'll be wanting to rework this system to be more compatible with a variety of customizations people use.

Reply:
I certainly underestimated the complexity a bit here. I might have a go at testing different randomization settings/games today just to make sure things work, if that doesn't work out I'll leave this alone for your expected rework.

Reply:
If you want to update the PR to exclude this section, that would work for me.